h/static/scripts/annotator/guest.coffee

```diff
@@ -71,6 +71,13 @@ module.exports = class Guest extends Annotator
     this._connectAnnotationSync(@crossframe)
     this._connectAnnotationUISync(@crossframe)
 
+    if @options.clickToClose
+      @wrapper = @element
+      .on 'click', (event) =>
+        if !@selectedTargets?.length
+          this.hideFrame()
+      this
```

Comment:
Oh, never mind. I see this is because `_setupWrapper` is gone entirely. That's no good. I think the base `Annotator` class calls `_setupWrapper`. It might be better if you keep this code in there, otherwise add `_setupWrapper` to the list of no-op functions that follows: `_setupDynamicStyle` and such.